src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { Errors } from '../../types/Errors';
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: Errors;
+  onErrorMessage: React.Dispatch<React.SetStateAction<Errors>>;
```

Comment:
use this syntax in all such cases(more readable)
```suggestion
  onErrorMessage: (error: Errors) => void;
```